src/northbridge/amd/pi/agesawrapper.c

```diff
@@ -302,7 +302,7 @@ AGESA_STATUS agesawrapper_amdinitlate(void)
 	Status = AmdInitLate(AmdLateParams);
 	if (Status != AGESA_SUCCESS) {
 		agesawrapper_amdreadeventlog(AmdLateParams->StdHeader.HeapStatus);
-		ASSERT(Status == AGESA_SUCCESS);
+		printk(BIOS_WARNING, "AmdInitLate returned error!\n");
```

Comment:
Yes, agree. In legacy this one is also asserting, but the assert was commented out. It looks like the AmdLateInit Agesa call returns correct parameters (Acpi tables), but it's asserting nevertheless. It could be due to a bug in Agesa, but as far as I've tried, we have only this one version which works.